src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/DeleteCommand.java

```diff
@@ -135,6 +135,14 @@ public void validate(String[] args) throws FailedValidationException {
             throw new FailedValidationException(errorMessage);
         }
 
+        for (String arg : args) {
+            if (arg == null) {
+                String errorMessage = "One or more arguments were null";
+                cw.println(errorMessage);
+                throw new FailedValidationException(errorMessage);
+            }
+        }
+
```

Comment:
This snippet looks like it's worth extracting to a utility method somewhere, rather than copy-pasting the implementation around.